### DISCUSSION TITLE
Why is `go_install` and `http` type packages not supported by `aqua rm` unless `--all` is passed?

### DISCUSSION THREAD

**Discussion by sapphi-red** [aquaproj/aqua#2903] [Deprecated (Question and Answer)] [answered]
### Question

I saw this limitation here.
https://aquaproj.github.io/docs/guides/uninstall-packages/#limitation

Is it just not implemented or is there a limitation somewhere?

### Background of the question

I was thinking if I should create a feature request for adding support for that.

**Comment by suzuki-shunsuke** [comment]
I don't remember why go_install packages can't be uninstalled without `--all` option, but I think we can support it.

On the other hand, to support uninstalling `http` type package, we need to add fields to registry settings.
Otherwise, aqua can't see which directories aqua should remove.

**Reply by sapphi-red** [reply]
Thanks for answering!

I see. Then, I'll create an issue about the go_install support. 👍 

Update: created https://github.com/aquaproj/aqua/issues/2905

**Comment by suzuki-shunsuke** [comment]
- https://github.com/aquaproj/aqua/pull/3101

**Reply by suzuki-shunsuke** [reply, marked as answer]
v2.33.0 is out 🎉
https://github.com/aquaproj/aqua/releases/tag/v2.33.0

This release supports not only `go_install` but also `http` packages.
Furthermore, we could resolve this issue without adding a new field.

https://github.com/aquaproj/aqua/issues/2905#issuecomment-2119018187

> To resolve this issue, my idea is to add a field to detect a file path to be removed.
